### PR TITLE
9.6 animatedbackground

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Application/src/visuals/Draw.java [hunk 1]
```diff
@@ -22,6 +23,9 @@
     private ArrayList<Message> messages = new ArrayList<>(); // Stores the messages between nodes.
     private int offset; // Used for message ordering
     private int class_size = 0; // Used for message positioning
+	
+    //stores an animated gif file specifically made for this application, which contains an 8-bit animation of a sky/ocean view
+    private static Image animatedBackground = new Image("resources/SkyGIF.gif");
```
Comment (Sebastian-Fransson) [inline Application/src/visuals/Draw.java:29]: Does this need to be static?

File: Application/src/visuals/Draw.java [hunk 2]
```diff
@@ -11,8 +12,8 @@
 import static visuals.DiagramView.tabPane;
 
 /**
- * @version 0.7
- * @author Pontus Laestadius, Sebastian Fransson
+ * @version 0.10
```
Comment (Sebastian-Fransson) [inline Application/src/visuals/Draw.java]: This is reverting the version.

**Review by Sebastian-Fransson — CHANGES_REQUESTED** [review]
Just small stuff

File: Application/src/visuals/Draw.java (hunk 1 above)
Comment (ghost) [reply Application/src/visuals/Draw.java:29]: in relation of memory and performance; it is static to be loaded only once in the canvas

File: Application/src/Parser.java [hunk 3]
```diff
@@ -7,7 +7,7 @@
```
Comment (pontuslaestadius) [inline Application/src/Parser.java:7]: If you want to change the version of something outside 9.6 make a seperate pull request.

File: Application/src/visuals/Draw.java (hunk 1 above)
Comment (pontuslaestadius) [reply Application/src/visuals/Draw.java:29]: ^ previous comment refers to what I said.

**Review by Sebastian-Fransson — APPROVED** [review]

File: Application/src/visuals/Draw.java [hunk 4]
```diff
@@ -11,8 +12,8 @@
 import static visuals.DiagramView.tabPane;
 
 /**
- * @version 0.7
- * @author Pontus Laestadius, Sebastian Fransson
```
Comment (pontuslaestadius) [inline Application/src/visuals/Draw.java]: Adding two lines does not warrent an author tag? Or does it?

**Review by pontuslaestadius — DISMISSED** [review]
Not a fan of the water color and I would like some mountains!

File: README.md [hunk 5]
```diff
@@ -1,7 +1,13 @@
-﻿# [Group 4, FUML: Making UML Fun]
-
+# [FUML: Making UML Fun]
+ 
+ <img width="200" height="200" src="https://github.com/timjon/group4/blob/develop/Application/src/resources/logo.png">
```
Comment (timjon) [inline README.md:3]: Whats this?

**Review by timjon — CHANGES_REQUESTED** [review]

File: Application/src/visuals/Draw.java (hunk 1 above)
Comment (ghost) [reply Application/src/visuals/Draw.java:29]: Congratulations

File: Application/src/visuals/Draw.java (hunk 4 above)
Comment (ghost) [reply Application/src/visuals/Draw.java]: I disagree
The task didnt have just 2 lines of effort

File: README.md (hunk 5 above)
Comment (ghost) [reply README.md:3]: Added logo to readme file

Comment (timjon) [reply README.md:3]: Maybe it should say that then

Comment (ghost) [reply README.md:3]: I can't add comments in the readme file, but if you view it on git you will see the logo

Comment (timjon) [reply README.md:3]: Oh i see

File: Application/src/visuals/Draw.java (hunk 4 above)
Comment (timjon) [reply Application/src/visuals/Draw.java]: The amount of effort shouldnt matter for this though, since its still 2 lines of code

Comment (timjon) [reply Application/src/visuals/Draw.java]: I suggest that you add yourself as a contributor or collaborator instead

Comment (kosara-g) [reply Application/src/visuals/Draw.java]: Can you do that? We haven't discussed adding these roles in the comments so far

File: README.md (hunk 5 above)
Comment (kosara-g) [reply README.md:3]: I'm confused, what changes were requested?

Comment (ghost) [reply README.md:3]: @gusgolko confused about the readme file? tim just had a question and i answered. if you are confused about the pull request, then all that i have to change is the author tag if we agree on that in the meeting. other than that #12 is read to be merged

**Comment by pontuslaestadius** [comment]
How does this interact when you don't animate the canvas? Does it stop the animation?

File: Application/src/visuals/Draw.java [hunk 6]
```diff
@@ -86,20 +90,16 @@ public void redraw() {
...
         gc.clearRect(0,0,getWidth(), getHeight()); // Clears the canvas
-
-        gc.setFill(Color.ALICEBLUE); // Sets the color to GREY
-        int split = getHeight()/2 +getHeight()/4;
-        gc.fillRect(0,0,getWidth(),split);
-        gc.setFill(Color.CORNFLOWERBLUE); // Sets the color to GREY
-        gc.fillRect(0,split,getWidth(), getHeight());
-        gc.setFill(Color.BLACK); // Resets the color to BLACK
+        // adds an animated gif file to the canvas with proper height and width
+        gc.drawImage(animatedBackground,10,10,this.canvas.getHeight()*2,this.canvas.getWidth()/1.5);
```
Comment (pontuslaestadius) [inline Application/src/visuals/Draw.java]: What does 'proper' mean? 1.5 and 2 times the size of the canvas?

**Review by timjon — APPROVED** [review]

**Review by kosara-g — APPROVED** [review]

**Pull request merged**